tests_e2e/test_e2e

```diff
@@ -0,0 +1,15 @@
+import os
+import pytest
+import requests
+from todo_app import app
+from dotenv import load_dotenv, find_dotenv
+from selenium import webdriver
+from selenium.webdriver.firefox.options import Options
+    
+@pytest.fixture
+def client():
+    file_path = find_dotenv('.env.test')
+    load_dotenv(file_path, override=True)
+    test_app = app.create_app()
+    with test_app.test_client() as client:
+        yield client
```

Comment:
I think this is actually the integration test style fixture rather than the end-to-end one, but appreciate this bit isn't really finished yet!